carina-core/pom.xml

```diff
@@ -96,9 +98,10 @@
 
 		<!-- Test utilities -->
 		<dependency>
-			<groupId>org.testng</groupId>
```

Comment:
removal of testng is incorrect. we can't reuse transitive dependency here and must use the version declared on our end! Please return!

Reply:
done